sql-cli/sql_cli/__main__.py

```diff
@@ -37,111 +41,105 @@ def about() -> None:
 
 @app.command(help="Generate the Airflow DAG from a directory of SQL files.")
 def generate(
-    directory: Path = typer.Argument(
+    workflow_name: str = typer.Argument(
```

Comment:
In future, depending on the feedback, we may want to have `dags_folder` as an option - so this works as an workflow export.